content/docs/command-reference/plots/diff.md

```diff
@@ -105,12 +105,21 @@ file:///Users/dmitry/src/plots/logs.html
 Compare two specific versions (commit hashes, tags, or branches):
 
 ```dvc
-$ dvc plots diff --targets logs.csv HEAD 0135527
+$ dvc plots diff HEAD 0135527 --targets logs.csv
 file:///Users/usr/src/plots/logs.csv.html
 ```
 
 ![](/img/plots_diff.svg)
 
+> Alternatively, you can also run above statement as:
+>
+> ```dvc
+> $ dvc plots diff --targets logs.csv -- HEAD 0135527
+> ```
+>
+> When you're specifying multiple revisions after `--targets`, use `--` so that
+> argument parser doesn't confuse them as options for `--targets` argument.
+
```

Comment:
Nice, I didn't even realize we can do that. But this doesn't close #1425

Reply:
hmm ... what else would expect here to be mentioned, @jorgeorpinel ?

Reply:
@shcheklein I think we should also include info that:
`dvc plots diff --targets t1.json t2.csv HEAD v1 v2` -> this is bad and won't work. This should be explicitly stated, as in most cases it feels natural to first provide options and only then the positional arguments.

Reply:
> We should include some note for dvc plots diff to hint possible use cases:
dvc plots diff HEAD v1 v2 --targets t1.json t2.csv
dvc plots diff --targets t1.json t2.csv -- HEAD v1 v2

And let's put it in the description please @imhardikj

Reply:
I think it's better to include this in the `--target` option description. I would try not to explode description too much with examples. At least that's not what we usually do.

Reply:
Good point. It's better to put it in --targets indeed. It's the first option listed, right after the Description anyway.

I have nothing against super short examples in the cmd ref descriptions though, we use them a little in run I think. It will also be "strange" to have it in the option bullets. But again, I agree this one should go in --target.

Reply:
I would inline and simplify them as much as possible. And would try to have a single example.